src/Draco.Compiler/Internal/Documentation/Extractors/MarkdownDocumentationExtractor.cs

```diff
@@ -0,0 +1,24 @@
+using Draco.Compiler.Internal.Symbols;
+
+namespace Draco.Compiler.Internal.Documentation.Extractors;
+
+/// <summary>
+/// Extracts markdown into <see cref="SymbolDocumentation"/>.
+/// </summary>
+internal sealed class MarkdownDocumentationExtractor
```

Comment:
If you have a pattern for a class, where the usage will always be `new MyClass(...).DoYourThing()`, always:
 * make the ctor private
 * Introduce a public static method that takes the ctor parameters, instantiates the class and call `DoYourThing`

We follow this with a bunch of classes.